src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs

```diff
@@ -409,15 +409,15 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(
 				new ContextMenuFlyoutItemViewModelBuilder(Commands.OpenFileLocation).Build(),
 				new ContextMenuFlyoutItemViewModelBuilder(Commands.OpenInNewTabAction)
 				{
-					IsVisible = UserSettingsService.GeneralSettingsService.ShowOpenInNewTab
+					IsVisible = UserSettingsService.GeneralSettingsService.ShowOpenInNewTab && Commands.OpenInNewTabAction.IsExecutable
```

Comment:
This does resolve the issue, but isn't `ContextMenuFlyoutItemViewModelBuilder` supposed to handle this?

Reply:
It seems like my comment was already addressed here https://github.com/files-community/Files/issues/16626#issuecomment-2557594864